targets/mgconvco/targets.go

```diff
@@ -0,0 +1,59 @@
+package mgconvco
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/go-logr/logr"
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+	"go.einride.tech/mage-tools/mglog"
+	"go.einride.tech/mage-tools/mgtool"
+)
+
+const version = "0.3.7"
+
+var executable string
+
+func ConvcoCheck(ctx context.Context, rev string) error {
+	logger := mglog.Logger("convco-check")
+	ctx = logr.NewContext(ctx, logger)
+	mg.CtxDeps(ctx, prepare)
+	logger.Info("checking...", "rev", rev)
+	return sh.RunV(executable, "check", rev)
```

Comment:
This is set on the prepare function but I don't see where it is called. How is the prepare function meant to be called?

Reply:
More specifically I meant the "executable" variable

Reply:
It’s called via the `mg.CtxDeps`

Reply:
Doh. Missed it somehow